### ISSUE TITLE
Customize names of ignore files (support .gitignore etc)

### ISSUE THREAD

**Issue by serhiy-storchaka** [neuromation/platform-client-python#1560]
Support of `.neuroignore` files was added in #1446.

We need to add support of other ignore files (e.g. `.gitignore`) and also an option to disable `.neuroignore` files.

The proposition is to add option `--exclude-from-files` to specify a space separated list of ignore files.

* `--exclude-from-files=".neuroignore .gitignore"` -- use both `.neuroignore` and `.gitignore`.
* `--exclude-from-files=.gitignore` -- use *only* `.gitignore`.
* `--exclude-from-files=` -- do not use any ignore files.

By default `.neuroignore` is used.

**Comment by serhiy-storchaka** [comment]
@asvetlov @yartem

**Comment by yartem** [comment]
Good idea to avoid duplication of `.neuroignore` and `.gitignore`! If we add the flag `neuro cp --exclude-from-files`, will the corresponding `.neuro.toml` entry be automatically supported? Similar to what we have to exclude:
```
[storage]
cp-exclude = ["*.log"]
```
